app/Module/GoogleMapsModule.php

```diff
@@ -2346,12 +2394,11 @@ private function printHowManyPeople($level, $parent) {
 	 * @param string   $linklevels
 	 */
 	private function printGoogleMapMarkers($place2, $level, $parent, $levelm, $linklevels) {
+		echo 'var icon_type = null;';
 		if (!$place2['lati'] || !$place2['long']) {
-			echo 'var icon_type = new google.maps.MarkerImage();';
```

Comment:
The changes in this function look suspicious.  i.e. it's not the same changes you were making elsewhere.  `icon_type` is becoming a string, not an object?

You've removed the declaration of `point`, but still use it in the next line.  Is this right?